up & going/ch2.md

```diff
@@ -236,13 +236,13 @@ Qualquer valor que não esteja nessa lista de "falsy", é considerado "truthy".
 * `{ }`, `{ a: 42 }` (objects)
 * `function foo() { .. }` (functions)
 
-É importante lembrar que um valor não-`boolean` segue a coerção como "truthy"/"falsy" apenas se ele for coergido para `boolean`. Não é difícil se confundir com uma situação onde parece que estamos coergindo um valor para um `boolean` quando na verdade não está.
+É importante lembrar que um valor não-`boolean` segue a coerção como "truthy"/"falsy" apenas se ele for coergido para `boolean`. Não é difícil se confundir com uma situação onde parece que estamos coergindo um valor para um `boolean` quando na verdade não estamos.
```

Comment:
apenas se ele for ~~coergido~~ **coagido** (...) parece que estamos ~~coergindo~~**coagindo**